quickstarts/manage-data/airflow-airbyte-dbt.md

```diff
@@ -0,0 +1,243 @@
+---
+sidebar_position: 5.5
+title: Automate Data Movement and Transformation with Airflow, Airbyte, and dbt in Teradata Vantage
+author: Vidhan Bhonsle
+email: vidhan.bhonsle@teradata.com
+page_last_update: October 4th, 2024
+description: Using Airflow to Orchestrate Airbyte and dbt for Teradata
+keywords: [data warehouses, airflow, teradata, vantage, orchestration, object storage, airbyte, enterprise analytics, elt, dbt.]
+---
+
+import ClearscapeDocsNote from '../_partials/vantage_clearscape_analytics.mdx'
+import CommunityLink from '../_partials/community_link.mdx'
+import ProfileTabs from '../_partials/tabsDBTProfiles.mdx'
+
+# Automate Data Movement and Transformation with Airflow, Airbyte, and dbt in Teradata Vantage
+
+## Overview
+
+This tutorial demonstrates how to leverage Apache Airflow as orchestration tool to automate the movement of data from a source to Teradata using Airbyte. Once the data is in Teradata, trigger dbt to perform transformations, ensuring the data is clean, reliable and ready for analysis.
+
+## Prerequisites
+
+* Access to a Teradata Vantage instance.
+
+    <ClearscapeDocsNote />
+
+* Python **3.10** or later installed.
+
+* [Docker and Docker Compose (Docker Desktop)](https://docs.docker.com/get-started/get-docker/)
+
+* [Airbyte command line tool](https://docs.airbyte.com/using-airbyte/getting-started/oss-quickstart)
+
+## Demo project setup
+
+1. Clone the tutorial repository and cd into the project directory:
+```bash
+git clone https://github.com/teradata/quickstarts.git
+cd quickstarts
+```
+
+2. Navigate to the directory:
+```bash
+cd airbyte_dbt_airflow_teradata
+```
+Open the code in your preferred IDE.
+
+
+## Setting up Teradata instance
+Follow the instructions from [Getting started with ClearScape Analytics Experience](https://developers.teradata.com/quickstarts/get-access-to-vantage/clearscape-analytics-experience/getting-started-with-csae/) and acquire following:
+
+*  Host
+* username
+* password
+
+Jot it down, it will be needed in next step.
+
+## Edit `profiles.yml` file
+Navigate to `dbt_project` folder, open `profiles.yml` file, it contains configuration for dbt to connect with your data platform. Update this file with your Teradata connection details:
+* Host (from previous step)
+* username (from previous step)
+* password (from previous step)
+* schema: `airbyte_td`
+
+And move the file in location shown below:
+
+<ProfileTabs/>
+
+## Airbyte setup
+
+1. After you install Airbyte OSS locally using `abctl`:
+
+```bash
+abctl local install
+```
```

Comment:
This didn't work for me because of https://github.com/airbytehq/airbyte/issues/46309 . Basically, Docker throttles anonymous pulls and abctl has a bug that doesn't allow users to pass Docker credentials to the pod that deploys connectors. I think we need to add an admonition that if a user encounters `429 Too Many Requests` error, then they should fall back to Docker Compose (and give the steps to do so).